### PR TITLE
feat: view test in leaner and submit test

### PR DESCRIPTION
**Pull request by anhtuyetdut**

### CODE REVIEW

**Review by huynhducthanhtuan — APPROVED** [review]

**Review by pv195 — APPROVED** [review]

**Review by pv195 — APPROVED** [review]

File: src/pages/Test/index.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+  const [results, setResults] = useState({});
+  const [checked, setChecked] = useState({});
+
+  const getTestDetail = () => {
+    getTestDetailApi(testId).then((data) => {
+      if (data.error) {
+        toast.error(data.message);
+      } else {
+        const test = data?.data;
```
Comment (phamvanhan68) [inline src/pages/Test/index.js:28]: const test = data?.data;
if there is no data.data, meaning that test = undefined, the rest of code will be error.
 
Also, data.data looks confused, should use **res** or **response** instead data

File: src/pages/Test/index.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+              <div className={styles.question_form}>
+                <div className={styles.question}>
+                  <span className={styles.content}>
+                    Question {index}. {question.content}
+                  </span>
+                  <span className={styles.score}>Score: {question.score}</span>
+                </div>
+                <div className={styles.answers}>
+                  {question.answers.map((answer) => (
+                    <>
```
Comment (phamvanhan68) [inline src/pages/Test/index.js:118]: it's fine to remove <> and </>

Comment (huynhducthanhtuan) [reply src/pages/Test/index.js:118]: OK. We used to Fragment instead. But it's a forgot moment. 😅

File: src/pages/Test/index.js (hunk 1 above)
Comment (huynhducthanhtuan) [reply src/pages/Test/index.js:28]: @anhtuyetdut

**Pull request merged**